test/processors/base_processor_test.rb

```diff
@@ -23,6 +21,42 @@ class BaseProcessorTest < ActiveSupport::TestCase
     }
   }
 
+  let(:audio_monster) do
+    Minitest::Mock.new
+  end
+
+  let(:processor) do
+    BaseProcessor.new(logger: Logger.new('/dev/null')).tap do |p|
+      p.audio_monster = audio_monster if travis?
+    end
+  end
+
+  let(:mp3_info) do
+    {
+      size: 774059,
+      content_type: 'audio/mpeg',
+      format: 'mp3',
+      channel_mode: 'Mono',
+      channels: 1,
+      bit_rate: 128,
+      length: 48.352653,
+      sample_rate: 44100
+    }
+  end
+
+  let(:ogg_info) do
+    {
+      size: 164196,
+      content_type: 'audio/ogg',
+      format: 'ogg',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.